cmd/osbuild-worker/jobimpl-depsolve.go

```diff
@@ -127,6 +134,13 @@ func (impl *DepsolveJobImpl) Run(job worker.Job) error {
 	}
 
 	var result worker.DepsolveJobResult
+	// ALWAYS return a result
+	defer func() {
+		err = job.Update(&result)
+		if err != nil {
+			logWithId.Errorf("Error reporting job result: %v", err)
+		}
+	}()
```

Comment:
Technically, this should go before line number 132. But it can be a follow-up.

Reply:
Crap. No, let's get it right first :)